src/engraving/dom/mscore.h

```diff
@@ -87,6 +87,9 @@ static constexpr int INVALID_FRET_INDEX   = -1; // no ordinal for a fret
 
 static constexpr ID INVALID_ID = 0;
 
+static constexpr double MAX_DOUBLE = std::numeric_limits<double>::max();
```

Comment:
Why not `DBL_MAX` and `INT_MAX`?

Reply:
Actually I agree with this now. Normally I would prefer writing `std::numeric_limits<double>::max()` because that's more idiomatic C++ than those C macros. But it's also a bit verbose, so therefore Michele made a constexpr for it. But that's hardly any better than using the C macros.

Reply:
Personally, I find it a bit more elegant to define our own constants, even though they are just basic redefinitions. We do it also for other stuff, like `mu::nidx` or `track_idx_t` or `staff_idx_t` etc. I'll admit this is a bit of a limit case, but it still cleans our code from ugly C macros and verbose std calls, so I'd prefer to keep it this way. It also leaves us more free to change how the constant is implemented (though I can't think of why we'd want to).

Reply:
Agree with @Jojo-Schmitz. It's better to use DBL_MAX. Also, mscore.h is not the best place for such constants (nor is shape.cpp, where you redefine it)